lms/djangoapps/instructor_task/tasks_helper/enrollments.py

```diff
@@ -196,15 +196,15 @@ def upload_students_csv(_xmodule_instance_args, _entry_id, course_id, task_input
     """
     start_time = time()
     start_date = datetime.now(UTC)
-    enrolled_students = CourseEnrollment.objects.users_enrolled_in(course_id)
+    enrolled_students = CourseEnrollment.objects.users_enrolled_in(course_id, only_active=task_input['only_active'])
     task_progress = TaskProgress(action_name, enrolled_students.count(), start_time)
 
     current_step = {'step': 'Calculating Profile Info'}
     task_progress.update_task_state(extra_meta=current_step)
 
     # compute the student features table and format it
-    query_features = task_input
-    student_data = enrolled_students_features(course_id, query_features)
+    query_features = task_input['query_features']
+    student_data = enrolled_students_features(course_id, query_features, only_active=task_input['only_active'])
```

Comment:
Nit: below, at line 219, maybe we should change the name of the resulting CSV file depending on whether it's active-only? Right now it comes out with `"student_profile_info"` either way. Can just do `csv_name = 'active_student_profile_info' if only_active else 'student_profile_info'`.

Reply:
Good point, I've included your suggested change.